bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
+
+FQDN=ovirt.cockpit.lan
+hostnamectl set-hostname ${FQDN}
+
+# Latest stable repos
+rpm -ivh http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```

Comment:
it has it installed on filesystem, but AFAIU it still needs to be imported into RPM. Perhaps it's missing from the libguests setup?